src/components/TextInput/TextInput.tsx

```diff
@@ -2,12 +2,15 @@ import React, { ChangeEvent, FunctionComponent } from 'react';
 
 import { Icon } from '../Icon/Icon';
 
+type Type = 'password' | 'text' | 'email';
```

Comment:
add 2 textboxes to the first story so we can test them out in the storybook
maybe add placeholder: "Enter your password", "Enter your email"

Reply:
Maybe we should add these options to this list as well, but we don't necessarily need to add them to the stories
We just add them, so that if we need any of them in the future, we don't need to update the component to be able to use them.
* color
* email
* number
* password
* range
* search
* tel
* text
* url

Reply:
@e2o @bertyhell Don't want to start a fire but you're both saying the opposite I think. 😅

Reply:
The types you added were different (hidden / submit / file / ...), I would agree on @bertyhell's extra types, -color and -range.

Reply:
Yes, mb.